src/Aspire.Hosting/Dashboard/DashboardServiceHost.cs

```diff
@@ -49,7 +49,7 @@ internal sealed class DashboardServiceHost : IHostedService
     public DashboardServiceHost(
         DistributedApplicationOptions options,
         DistributedApplicationModel applicationModel,
-        IKubernetesService kubernetesService,
+        ApplicationExecutor applicationExecutor,
```

Comment:
Why?

Reply:
So we can call start/stop on the executor from the commands.

Reply:
I've thought about this more, and the issue is the ServiceProvider that is passed into the command callback when it executes. It has been the dashboard host's service provider. I think that's wrong, and it should be the app host's service provider.

Using the app host service locator means services registered in the app host by user code are available in registered commands. It also makes ApplicationExecutor available without having to pass it into the dashboard service host.

Reply:
Fixed.